src/patternfly/components/NotificationBadge/notification-badge.scss

```diff
@@ -1,35 +1,48 @@
 .pf-c-notification-badge {
-  --pf-c-notification-badge--after--BorderColor: var(--pf-global--BackgroundColor--100);
-  --pf-c-notification-badge--after--BorderRadius: var(--pf-global--BorderRadius--lg);
+  --pf-c-notification-badge--Color: var(--pf-global--Color--light-100);
```

Comment:
The changes in this PR assume the notification badge will always be used against a dark background. The previous notification badge worked in both light and dark, and got it's dark theme styles by adopting the global var changes we apply with the dark theme in `.pf-c-page__header`.

It seems like we can make this component work in both a dark and light theme, if we limit this color change to the unread and attention states. Then the normal/read state will inherit color, or we could set it to `--pf-c-notification-drawer--Color: var(--pf-global--Color--100);`, though that doesn't seem necessary.

Then if we want to include dark theme examples on this page, you could add `.pf-t-dark` to the example wrapper, and can even set it's background to `var(--pf-global--BackgroundColor--100)`, and it will change to `#151515`, and it will mimic use like in the page header.

wdyt?